src/main/java/com/tabnine/general/StaticConfig.java

```diff
@@ -42,15 +42,26 @@
 
     public static final Icon ICON = IconLoader.findIcon("/icons/tabnine-icon-13px.png");
 
+
     public static String getServerUrl() {
         return Optional.ofNullable(System.getProperty(REMOTE_BASE_URL_PROPERTY)).orElse("https://update.tabnine.com");
     }
 
+    // https://update.tabnine.com/bundles/version
+    public static String getBundleServerUrl() {
+        return Optional.ofNullable(System.getProperty(REMOTE_BASE_URL_PROPERTY)).orElse("https://update.tabnine.com/bundles");
+    }
+
     @NotNull
     public static String getTabNineVersionUrl() {
         return Optional.ofNullable(System.getProperty(REMOTE_VERSION_URL_PROPERTY)).orElse(getServerUrl() + "/version");
     }
 
+    @NotNull
+    public static String getTabNineBundleVersionUrl() {
+        return Optional.ofNullable(System.getProperty(REMOTE_VERSION_URL_PROPERTY)).orElse(getBundleServerUrl() + "/version");
```

Comment:
This will resolve `https://update.tabnine.com/bundles/version`, is that the intended url?

Reply:
yep